geoserver/jetty-env.xml

```diff
@@ -35,5 +35,23 @@
        </New>
      </Arg>
    </New>
+   <New id="geoserver-gwc" class="org.eclipse.jetty.plus.jndi.Resource">
+     <Arg>jdbc/gwc</Arg>
+     <Arg>
+       <New class="org.apache.commons.dbcp.BasicDataSource">
+         <Set name="driverClassName">org.postgresql.Driver</Set>
+         <Set name="username"><Env name="JNDI_GEORCHESTRA_JDBC_USERNAME"/></Set>
```

Comment:
The `JNDI_GEORCHESTRA_JDBC_` prefix is not explicit enough IMO.
Here we are configuring a db connection pool for geowebcache to store its cache usage statistics.

Something like `GWC_JNDI_PGUSER`, `GWC_JNDI_PGPASSWORD`, `GWC_JNDI__PGHOST`, `GWC_JNDI_PGPORT`, `GWC_JNDI_PGDATABASE` etc ... would suit better IMO.

Reply:
okay